### DISCUSSION TITLE
What is the best way to handle multiple prisma schemas?

### DISCUSSION THREAD

**Discussion by jakeleventhal** [prisma/prisma#18218] [Q&A] [answered]
I have a monorepo with pnpm. I have a schema and a database that works fine. I am able to import types for models like so
```typescript
import { UserModel } from '@prisma/client';
```

However, i want to create a new DB schema for a separate project in the same repo. How should that be handled? I want to be able to import '@prisma/client' and not have overlap on my schema models

**Comment by nurul3101** [comment, marked as answer]
Hello @jakeleventhal 👋 

You can define two different custom paths for the generated PrismaClients and can then import them as needed.
Here's a [reference](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#using-a-custom-output-path) on how you can define custom output path in the `schema.prisma` file.

**Reply by nurul3101** [reply]
@jakeleventhal If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
